### DISCUSSION TITLE
Support for :enabled selector

### DISCUSSION THREAD

**Discussion by justusip** [tailwindlabs/tailwindcss#7903] [Ideas]
Is there any plan for adding the :enabled pseudo class? I would like to add style to my button only if it is enabled and active (e.g. :enabled:active:class_to_add). If I only use :active, the class can be activated even if the button is disabled.

[https://developer.mozilla.org/en-US/docs/Web/CSS/:enabled](https://developer.mozilla.org/en-US/docs/Web/CSS/:enabled)

**Comment by adamwathan** [comment]
Good suggestion! I've opened a PR here: https://github.com/tailwindlabs/tailwindcss/pull/7905
